Added human spaceport hint for Sestor fleet

MessyMix opened the pull request:
I, and apparently several others ( http://steamcommunity.com/home#scrollTop=0 ) have found difficulty in finding the Sestor when searching for them initially in human space. I just glossed over the actual system, because it is south of all the other pirate systems, which I checked. I thought that Farpoint would be the location of the Sestor fleet, so I was surprised when they did not turn up there. I actually had to resort to checking the save files to find the location of the fleet.

So I made a solution (took me a while) that involves talking to a dude who narrowly escaped the Sestor as two 53's jumped in while scouting human space. It's written in the style of the rest of the game, gives the player ample choices (I refrained from deciding actions _for_ the player; I gave him/her multiple choices), and nudges the player back in the direction of the pirate colonies. It will be super helpful for players like me who assumed that they had spread across the rest of human space beyond the pirate colonies of the northwest. This extra mission will just hint the player back in that direction.

What d'yall think? Feedback and everything welcome and appreciated. :D

Amazinite commented:
I think it makes sense to only have this offer in the north, as I don't imagine a Behemoth running all the way to the south, and it would be especially unbelievable if the player has a Jump Drive and some Ramscoops, therefore being able to make it to the south far faster than a Behemoth could.

MessyMix commented:
@Amazinite The thing is, later on Admiral Danforth is involved in the story, and the player informs Danforth of a Sestor fleet; Danforth puts 2 and 2 together and figures out that the Alphas are controlling the Sestor. I noticed this and didn't want to interfere with the rest of the mission chain, so I didn't involve Danforth in this one.

I will, however, relegate the mission offerings to near Farpoint.

jafdy commented:
Have him (the Behemoth pilot) tell a desk clerk at the Navy base. This lets him be told nothing came past Farpoint and we can assume that the clerk didn't bother telling anyone else about it.

MessyMix commented:
@jafdy so will it be offered at a spaceport, then? I don't think it will be a conversation that can be heard upon landing, but all the other conversations have a `landing` tag.

jafdy commented:
No, he tells you that he has talked to a desk clerk when he was at Farpoint.

MessyMix commented:
@jafdy sure. I will add that when I get the time.

In general, though, is this a good idea, do you guys think?

jafdy commented:
Yes! Extra hints can't really be a bad thing.

Pointedstick commented on data/wanderers middle.txt:
```diff
@@ -1766,7 +1766,51 @@ mission "Wanderers: Sestor Search: Varu K'prai"
+				`	(Ignore him.)
+					decline
+			label approach
+			`	You get a better view of the Behemoth——the "Heart of Gold", as indicated on the ship's stern——as you walk up to it. The rear of the ship is badly hammered, and black soot-like stains streak forward across the hull. You can't help but notice that at several projectile entry along the hull, the metal is bent smoothly inwards, almost as if the penetrator had seared or melted its way through the ...
```
Grammar error: "at several projectile entry along the hull"

Bladewood replied:
https://github.com/endless-sky/endless-sky/blob/f655f242a72041a547dd43d52112bba2688fae1c/data/kestrel.txt#L129 is the only other instance where `--` is used in text. Are we sure we want to do that here?

Bladewood replied:
Also, this needs a closing backtick.

Pointedstick commented on data/wanderers middle.txt:
```diff
@@ -1766,7 +1766,51 @@ mission "Wanderers: Sestor Search: Varu K'prai"
+				`	"Where did this fight take place?"`
+				`	"You should be glad you're still in one piece."
+					goto glad
+			`	"Well," the pilot continues, "we had hired a beefy Leviathan escort to Farpoint in the Alnitak system. When we were going in for landing, two of these hostile ships jumped in from the northwest, the direction of all the anarchist colonies around here. The ships weren't big, but they were tough——our four heavy laser turrets weren't enough to produce even a dent in their shields. They immediat ...
```
"They immediately shredded the Leviathan warship"

How about just "They shredded the Leviathan"?

Pointedstick commented on data/wanderers middle.txt:
```diff
@@ -1766,7 +1766,51 @@ mission "Wanderers: Sestor Search: Varu K'prai"
+			`	"No problem, kiddo," the captain says, although he looks awed that you are even attempting to tackle such powerful ships. "Remember, northwest of Farpoint. If they're anarchists of any sort, they'll be hiding in one of those pirate systems up there. Take care of yourself out there; those are some dangerous ships that jumped us." With that, he bids you farewell.`
+					decline
+			label glad
+			`	"Yeah, those ships would have obliterated our Heart of Gold if we hadn't jumped pronto." The pilot looks at you, then asks quizzically, "Say, do you know anything about those aggressors?"`
```
"our Heart of Gold" sounds weird to me. How about "us" or "our ship"?

"aggressors" seems out of place here. Maybe "that fleet" instead?

Bladewood replied:
I agree with the former; the latter is purely opinion, but it may seem too formal compared to what the captain has already said.

Pointedstick commented on data/wanderers middle.txt:
```diff
@@ -1766,7 +1766,51 @@ mission "Wanderers: Sestor Search: Varu K'prai"
+			`	The pilot seems to display an expression of distaste upon hearing of aliens. You try to describe the dynamic between the exiles, the Efreti, Mereti, and Sestor, but to no avail whatsoever. The pilot quickly loses interest in your explanation, so you decide to steer the conversation to an end. "Thank you for your time," you say to the captain. The pilot quickly perks up again, and says, "No p ...
+				decline
+			label no
+			`	The pilot seems to be disappointed in your response, as if he wanted to learn more of the mysterious ships that attacked the Heart of Gold, so you decide to steer the conversation to an end."Thank you for your time," you say to the captain. The pilot quickly perks up again, and says, "No problem, kiddo. Remember, northwest of Farpoint. If they're anarchists of any sort, they'll be hiding in  ...
```
The text that begins "The pilot quickly perks up again..." could probably be broken out into its own block to avoid having to repeat it.

KiLEdEnNis replied:
> the conversation to an end."Thank you for your time,"

add a space between end. and "Thank

MessyMix commented:
@Pointedstick, thanks for the feedback. I'll incorporate your changes soon.

Bladewood commented on data/wanderers middle.txt:
```diff
@@ -1766,7 +1766,51 @@ mission "Wanderers: Sestor Search: Varu K'prai"
+		conversation
+			`As you bring your ship in for landing at an empty pad, you notice the captain of an adjacent Behemoth inspecting his ship, which is raked with projectile punctures.`
+			choice
+				`	(Approach the captain and try to find out what happened)`
```
Should be a period before the closing parenthesis.

Bladewood commented on data/wanderers middle.txt:
```diff
@@ -1766,7 +1766,51 @@ mission "Wanderers: Sestor Search: Varu K'prai"
+			`	You get a better view of the Behemoth——the "Heart of Gold", as indicated on the ship's stern——as you walk up to it. The rear of the ship is badly hammered, and black soot-like stains streak forward across the hull. You can't help but notice that at several projectile entry along the hull, the metal is bent smoothly inwards, almost as if the penetrator had seared or melted its way through the ...
+			choice
+				`	"How did your ship end up in such a state?"`
+				`	"Did those damn pirates maul this freighter to pieces?"`
```
I'm pretty sure that Endless Sky choices are supposed to be less "aggressive".

KiLEdEnNis replied:
maybe it's worth to mention Danforth himself did swear too (but that was about a planetary bomb)
the player itself shouldn't be "forced" to swear in conversations

MessyMix replied:
I originally had "pesky", but that was weak. I'll change it somehow.

KiLEdEnNis replied:
yes but if you want to steer the convo in a certain conversation you would be forced to swear
either have neutral options to different branches or different "attitude" answers that have the same result (for some flavor)
my opinion though

MessyMix replied:
Point taken, good proposition (although I have a feeling that it would be redundant to just have an option to make the player swear).

KiLEdEnNis replied:
same, these messages seem fine if the 2nd one was "Did those pirates [maul this freighter to pieces?]" where I'd like a rewording between [ & ] but it'd be a lot better

Bladewood commented on data/wanderers middle.txt:
```diff
@@ -1766,7 +1766,51 @@ mission "Wanderers: Sestor Search: Varu K'prai"
+			choice
+				`	"How many ships were there? What did they look like?"`
+				`	"Where did this fight take place?"`
+				`	"You should be glad you're still in one piece."
```
Needs a closing backtick.

Bladewood commented on data/wanderers middle.txt:
```diff
@@ -1766,7 +1766,51 @@ mission "Wanderers: Sestor Search: Varu K'prai"
+			label glad
+			`	"Yeah, those ships would have obliterated our Heart of Gold if we hadn't jumped pronto." The pilot looks at you, then asks quizzically, "Say, do you know anything about those aggressors?"`
+			choice
+				`	"Yeah, those were Korath ships that jumped you."`
```
plzrewrite

MessyMix replied:
plzexplain

Bladewood replied:
"that jumped you"

Should be [politer, more formal, less colloquial].

MessyMix replied:
Really?

It was colloquial on purpose. Why should it be formal? I will change it, but I want to know why.

Bladewood replied:
[If it shows up at Urban Dictionary and nowhere else in an "official" capacity, it's probably not a good idea to use it.](https://duckduckgo.com/?q=to+jump+someone&t=ffab&ia=web)

MessyMix replied:
I realize it's informal and I'll switch it out, but I don't see the problem with using informal language.

As for your "nowhere else in an 'official' capacity", try the Oxford Dictionary of English. Official enough? Informal, yes, but wholly recognized.
![image](https://cloud.githubusercontent.com/assets/20450076/23679083/a650ea6a-0354-11e7-8dae-c782b337a548.png)

Amazinite replied:
@Bladewood I think that "getting jumped" is pretty common English, at least in the parts of the US that I've been to. Not sure about across the pond.

Lineth replied:
It's quite common English in the US. I've heard it p. much everywhere.

Bladewood replied:
*happily points out that it says "jump on"*
@Amazinite it is over here too.
But I don't think it's fitting in something like Endless Sky, that's all.

MessyMix commented:
Fixed, boys and girls. 🎉

Pointedstick commented on data/wanderers middle.txt:
```diff
@@ -1766,7 +1766,73 @@ mission "Wanderers: Sestor Search: Varu K'prai"
+			`	You get a better view of the Behemoth -- the "Heart of Gold", as indicated on the ship's stern -- as you walk up to it. The rear of the ship is badly hammered, and black soot-like stains streak forward across the hull. You can't help but notice that at several projectile entry points along the hull, the metal is bent smoothly inwards, almost as if the penetrator had seared or melted its way  ...
+			choice
+				`	"How did your ship end up in such a state?"`
+				`	"Was this beauty a victim of a pirate attack?"`
```
"the victim of a pirate attack" sound better to me than "a victim of a pirate attack" (two "a"s in quick succession are awkward).

MessyMix replied:
Mmkay.

Pointedstick commented on data/wanderers middle.txt:
```diff
@@ -1766,7 +1766,73 @@ mission "Wanderers: Sestor Search: Varu K'prai"
+					decline
+			
+			label approach
+			`	You get a better view of the Behemoth -- the "Heart of Gold", as indicated on the ship's stern -- as you walk up to it. The rear of the ship is badly hammered, and black soot-like stains streak forward across the hull. You can't help but notice that at several projectile entry points along the hull, the metal is bent smoothly inwards, almost as if the penetrator had seared or melted its way  ...
```
I don't like "black soot-like stains". At the minimum, "soot-like black stains" seems more natural, but I'm not sure "soot-like" is the right adjective here.

MessyMix replied:
Okay. I'm trying to get this effect, and I don't quite know how to describe it.

![image](https://cloud.githubusercontent.com/assets/20450076/24067957/7f0e8a94-0bbe-11e7-855c-709a6f0ce6a6.png)

MessyMix replied:
Scorch marks. Got it.

MessyMix replied:
Also, just a side note—this is meant to hint at the Korath Repeater Turrets which have heat damage.

Pointedstick replied:
I'm not sure anyone will manage to figure that out, but it's good for flavor!

Pointedstick commented on data/wanderers middle.txt:
```diff
@@ -1766,7 +1766,73 @@ mission "Wanderers: Sestor Search: Varu K'prai"
+				`	"I don't want to jump to any conclusions."`
+					goto no
+					
+			`	The pilot seems to display an expression of distaste upon hearing of aliens. You try to describe the dynamic between the exiles, the Efreti, Mereti, and Sestor, but to no avail whatsoever. The pilot quickly loses interest in your explanation, so you decide to steer the conversation to an end.`
```
I'm not sure what this is trying to communicate to the player. That the captain is racist or something? Why wouldn't he be interested in the fact that his ship was attacked by incredibly powerful aliens?

MessyMix replied:
Perhaps he's disgusted that there are new aliens in human space! That would be a major point!

[My headcanon had the Captain be extremely experienced in the far north, so Hai. Had a scuffle with the Unfettered, hated aliens ever since. It's just a nuance that is up for the player to decide. I never hint at anything, so what this is trying to communicate is open to interpretation. Is he a xenophobe (literally)? Does he have bad memories of them? Is he disgusted that there are new aliens in human space?]

Pointedstick replied:
Being disgusted and losing interest are totally different things, though. If he's a xenophobe, that's fine, but then why would he lose interest? He should get angry and start ranting about aliens or something

MessyMix replied:
Okay. I'll think about how to change it.

Pointedstick commented on data/wanderers middle.txt:
```diff
@@ -1766,7 +1766,73 @@ mission "Wanderers: Sestor Search: Varu K'prai"
+				goto end
+				
+			label end
+			`	"Thank you for your time," you say to the captain. The pilot quickly perks up again, and says, "No problem, kiddo. Remember, northwest of Farpoint. If they're anarchists of any sort, they'll be hiding in one of those pirate systems up there. Take care of yourself out there; if they really are alien ships, they must be pretty dangerous." With that, he bids you farewell.`
```
"Kiddo" is a pretty demeaning thing to call an adult. Are we supposed to think the Behemoth captain is a condescending jerk?

Amazinite replied:
https://github.com/endless-sky/endless-sky/wiki/QualityChecklist 
>Conversations should not imply that the player is a certain gender or use gendered pronouns for the player (call them "kid" or "Captain" or `<first>` instead).

KiLEdEnNis replied:
if everything becomes neutral it becomes too robotic, a kiddo once in a while won't hurt the game

Bladewood replied:
@Amazinite to be fair, she's a free worlds officer, which is higher than a mere merchant captain, so she has the power and authority and stuff to call you a kid.

KiLEdEnNis replied:
20-year-old FW officer calls 40-year-old kiddo
doesn't fit in most cases

Bladewood replied:
you're not expected to be over 40 years old by the time you join the free worlds...

Actually, how old IS the player when they start the game? Or is it not specified, left to be filled in, like many other details?

Amazinite replied:
The player started working in a textile factory at 15 to save up for a pilot license. Go ahead and count from there.

KiLEdEnNis replied:
the 20 & 40 years was an example; it generally doesn't fit, early 20s etc.? not imo

MessyMix replied:
It's almost a valid point, but as @Amazinite points out, it's used in the in-game text already. And also,  If James calls you a "kid", would you find him to be a condescending jerk?

If your answer was yes, well, he _does_ do so. "There's no quick road to riches, kid."  The Behemoth captain is meant to be an old-timer who is experienced at the game, as demonstrated by his witty decision to jump instead of land. Almost like a James that hasn't quite yet reached retirement. I personally don't think it's condescending, and didn't intend it to be so.

Bladewood replied:
> The Behemoth captain is meant to be an old-timer who is experienced at the game, as demonstrated by his witty decision to jump instead of land.

Bloody brilliant.

MessyMix replied:
@Bladewood can't tell if that's sarcastic or not.

MessyMix commented:
Incorporated some of your changes, @Pointedstick.

MessyMix commented:
Does this look good as of now? I think I addressed everything in this PR so far.

KiLEdEnNis commented:
wait you can credit yourself in the files? *edits PR*

Amazinite commented:
I don't think so, because no other file is done that way.

ph2000bis commented on data/wanderers middle.txt:
```diff
@@ -1766,7 +1766,73 @@ mission "Wanderers: Sestor Search: Varu K'prai"
+					decline
+					
+			label glad
+			`	"Yeah, those ships would have obliterated us if we hadn't jumped pronto." The pilot looks at you, then asks quizzically, "Say, do you know anything about those ships?"`
```
Are you sure it is worth using "quizzically"? I'm pretty sure not many people, starting with me, know this word, and [Dictionary.com](http://www.dictionary.com/browse/quizzically?s=ts) agrees.

Bladewood replied:
I'm fine with it.

MessyMix commented:
You can credit yourself for original content,  @Amazinite and Kile. Look at
syndicate world jobs, @Pointedstick credited himself in that one.

On Mar 20, 2017 03:26, "Bladewood" <notifications@github.com> wrote:

> *@Bladewood* commented on this pull request.
> ------------------------------
>
> In data/wanderers middle.txt
> <https://github.com/endless-sky/endless-sky/pull/2221#discussion_r106815858>
> :
>
> > +					goto thanks
> +					
> +			label where
> +			`	"Well," the pilot explains, "we were doing a large cargo run to Farpoint in the Alnitak system, when two of these hostile ships jumped in from the northwest, the direction of all the anarchist colonies in this area. I had to jump the ship in atmosphere to stop us from getting shredded by their strange melting weaponry. I immediately reported the attack to a desk clerk at the Navy base in the neighboring system Saiph, but their servers show no logs of any unidentified ships beyond Farpoint, so I guess they turned around after scouting out the Alnitak system."`
> +			choice
> +				`	"You're lucky as hell to be here today."`
> +					goto glad
> +				`	"Thanks for the information. I think that these hostile ships are the ones I'm hunting for."`
> +					goto thanks
> +			
> +			label thanks
> +			`	"No problem, kiddo," the captain says, although he looks awed that you are even attempting to fight such powerful ships. "Remember, northwest of Farpoint. If they're anarchists of any sort, they'll be hiding in one of those pirate systems up there. Take care of yourself out there; those are some dangerous ships that jumped us." With that, he bids you farewell.`
> +					decline
> +					
> +			label glad
> +			`	"Yeah, those ships would have obliterated us if we hadn't jumped pronto." The pilot looks at you, then asks quizzically, "Say, do you know anything about those ships?"`
>
> I'm fine with it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/endless-sky/endless-sky/pull/2221#discussion_r106815858>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATgLHNzskfbA6gBSCMYQhn0ZEi028Cd6ks5rnYFvgaJpZM4MTTYu>
> .
>

MessyMix commented:
Everyone, discussion over attribution of original content is over; consensus: let MZ decide.

I would like to point out that this is my work, and those _paragraphs_ up there are my creative writing. Anyone wanting to further the argument can take it to the [Discord](https://discord.gg/B5yueeQ)

My discussion with Wrzl was constructive. "we could in future, if it is not a complete file, mark sections with comments similar how the images are named"

MessyMix commented:
Everyone, the discussion regarding the attribution of credit is over; consensus: let MZ decide.

Just note that this is my own original work, and those paragraphs up there are my creative writing.

I have a lot left to say, but again—case closed. Anyone wanting to further the argument can take it to the [Discord](https://discord.gg/u6KepgG).

My discussion with Wrzl was constructive. Conclusion was that "we could in future, if it is not a complete file, mark sections with comments similar how the images are named".

Regardless, if y'all think this is ready, that's great. We'll see what MZ says about credit attribution.

KiLEdEnNis commented:
also from the Discord: updating credits.txt or making a 2nd one (the current one being the one in the moving list when ya launch the game, one with the contributors and what they did)
I'm making an issue on that so that'll fix this credit thing if MZ agrees

MessyMix commented:
Cleaned up some stuff. @Pointedstick now the pilot rants a little about the Hai :) even though I firmly believe that using "kid" isn't demeaning, I removed it because it would be out of character after his little rant. Thanks for your continued input.

KiLEdEnNis commented on data/wanderers middle.txt:
```diff
@@ -1823,15 +1823,15 @@ mission "Wanderers: Sestor Search: Human Spaceport"
 					goto no
 					
-			`	The pilot seems to display an expression of distaste upon hearing of aliens. You try to describe the dynamic between the exiles, the Efreti, Mereti, and Sestor, but to no avail whatsoever. The pilot quickly loses interest in your explanation, so you decide to steer the conversation to an end.`
+			`	"Aliens!?" The pilot's face reddens, and his eyes narrow in anger. "Like those detestable squirrles, you mean? Told me they were peaceful, went further north, and nearly had my ship torn to pieces!" You try to tell him of the dynamic between the exiles, the Efreti, Mereti, and Sestor, but he clearly does not want to hear about the Kor, so you decide to steer the conversation to an end.`
```
should have said only the Unfettered are mean to calm him down

Bladewood replied:
I personally like the old one more.

also, squirrels*

MessyMix replied:
@Bladewood, I'll make a compromise between yours and @Pointedstick's version. He won't lose interest, but he won't rant either. Thanks for the good catch.

Pointedstick replied:
I like the one one much better!

MessyMix replied:
The one one?

MessyMix commented:
@Pointedstick @Bladewood I'm torn! D:

Bladewood commented:
https://www.explainxkcd.com/wiki/index.php/Category:Compromise

MessyMix commented:
Anyone have any extra feedback? Suggestions? Thoughts? (looking at you MZ xD)

Amazinite commented:
Maybe not claim the file as yours?
*he said, poking the bear in the eye*

MessyMix commented:
@Amazinite 
>Everyone, the discussion regarding the attribution of credit is over; consensus: let MZ decide.

MessyMix commented:
There, okay? :3

Amazinite commented:
👌

MessyMix commented:
@endless-sky would love some feedback. ;)

MessyMix commented:
@endless-sky if it's a no, at least say directly; reticence is often unproductive. If it really is a no, I'd like to know why, and what I can do to fix it.

Zireael07 commented:
I believe endless sky is on vacation this month.

jafdy commented:
He's back. He has been back for a while.

endless-sky commented on data/wanderers middle.txt:
```diff
@@ -1766,7 +1766,73 @@ mission "Wanderers: Sestor Search: Varu K'prai"
+					decline
+			
+			label approach
+			`	You get a better view of the Behemoth -- the "Heart of Gold", as indicated on the ship's stern -- as you walk up to it. The rear of the ship is badly hammered, and black scorch marks streak forward across the hull. You can't help but notice that at several projectile entry points along the hull, the metal is bent smoothly inwards, almost as if the penetrator had seared or melted its way thro ...
```
To match the existing mission text, you should use single dashes instead of double dashes here.

endless-sky commented:
I had 136 new GitHub threads waiting for me when I got back from vacation, not to mention all my real-world obligations. So, getting caught up may take a while.

MessyMix commented:
@endless-sky no problem; just wanted to make sure that you weren't ignoring my PR intentionally. =D

MessyMix commented:
@endless-sky, incorporated your style fix. Does the rest of it look good?

MessyMix commented:
@endless-sky how's it looking? :D

MessyMix commented:
@endless-sky just so you know that this still exists. I know you're busy, so take your time—just so this doesn't fall into oblivion.

endless-sky commented on data/wanderers middle.txt:
```diff
@@ -1766,7 +1766,82 @@ mission "Wanderers: Sestor Search: Varu K'prai"
 			`	The Wanderers don't know what happened next, but the drones are no longer here, so they must have moved on to a different system.`
 				decline
 
-
```
Keep three blank lines between missions for the sake of readability, please (so if scanning quickly through the file, it's easy to see where each mission definition begins and ends).

endless-sky commented on data/wanderers middle.txt:
```diff
@@ -1766,7 +1766,82 @@ mission "Wanderers: Sestor Search: Varu K'prai"
+				`	"I don't want to jump to any conclusions."`
+					goto no
+					
+			`	The pilot seems to display an expression of distaste upon hearing of aliens. You try to describe the dynamic between the exiles, the Efreti, Mereti, and Sestor, but he clearly does not want to hear any more, so you decide to steer the conversation to an end.`
```
"The pilot seems to display an expression of distaste" is an awkwardly roundabout phrase - maybe just say, "The pilot frowns" or "The pilot seems taken aback by your casual mention of aliens."

MessyMix commented:
#2427 @endless-sky fixed your changes.
